fin/Dockerfile

```diff
@@ -74,21 +74,42 @@ RUN pip install https://github.com/pfultz2/cget/archive/57b3289000fcdb3b7e424c60
 # Install rclone
 RUN pip install https://github.com/pfultz2/rclone/archive/master.tar.gz
 
-# Install hcc from ROCm 3.0
-RUN rclone -b roc-3.0.x -c 286651a04d9c3a8e3052dd84b1822985498cd27d https://github.com/RadeonOpenCompute/hcc.git /hcc
-RUN LDFLAGS=-fuse-ld=gold cget -p $PREFIX install hcc,/hcc  && rm -rf /hcc
-
 # Make sure /opt/rcom is in the paths
 ENV PATH="/opt/rocm:${PATH}"
+# Install MIOpen
+ARG MIOPEN_DIR=/root/dMIOpen
+#Clone MIOpen
+RUN git clone https://github.com/ROCmSoftwarePlatform/MIOpen.git $MIOPEN_DIR
```

Comment:
@averinevg Fin is its own repository living separate from MIOpen. Fin extends MIOpen with additional functionality. Everything inside the fin/ folder is actually a git subtree. When building the docker for fin to run an MIOpen install is required, so it is cloned here.

Reply:
@cderb Since MIOpen does not depend on Fin, would not it be better to separate them?

Reply:
Somewhere there was a conversation that Fin should be included as a submodule to MIOpen so that there could be some insurance in the CI that updates to MIOpen did not break Fin.

Reply:
@junliume @JehandadKhan 
If it should be like a submodule why it's look like a code copy?